pages/vi/gitrepositories.md

```diff
@@ -38,7 +38,7 @@ The **upstream repository** is the one we are contributing to.
 3. `git remote add upstream https://github.com/treehouses/treehouses.github.io.git`
 4. `git remote -v` origins should remain the same, but you should also be fetching and pushing to OLE as the upstream now
 
-#### Sync Your Fork
+#### Sync/Update Your Fork
```

Comment:
I think you have solved this in the another PR, remove this so one RP for one issue

Reply:
Done